cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -57,10 +57,10 @@ else
   read -ra KUBECTL_PRUNE_WHITELIST <<< "${KUBECTL_PRUNE_WHITELIST_OVERRIDE}"
 fi
 
-ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-60}
+export ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-60}
```

Comment:
I actually added the export because shellcheck noticed that this variable wasn't used in this file, and so should either be deleted or exported. The export doesn't have any effect other than making the variable available to sub-processes, which shouldn't have any effect here.

It might be more readable to just `# shellcheck disable` for those variables.